swiss/src/app/question-form/question-form.component.ts

```diff
@@ -17,19 +18,32 @@ export class QuestionFormComponent implements OnInit {
   constructor(private fb: FormBuilder, private db: AngularFireDatabase, private dialog: MatDialog) { }
 
   questionForm: FormGroup;
-
   state: string;
-
   questionId: string;
 
   @Input()
   path: string
 
+  @Input()
+  delegates: Observable<ValueName[]>
+
+  @Input()
+  rounds: Observable<ValueName[]>
+
   answerPaths: Observable<string[]>
 
   ngOnInit() {
+    // TODO delegateId and roundId to names
```

Comment:
Last thing to do to display rounds and delegate IDs as names. But I could not find how to do it with the GroupForm & Firebase.